native/dlahqr.go

```diff
@@ -14,7 +14,7 @@ import (
 // upper Hessenberg matrix H, using the double-shift/single-shift QR algorithm.
 //
 // h and ldh represent the matrix H. Its contents on return depends on the value
-// of info. Dlahqr works primarily with the Hessenberg submatrix
+// of ok. Dlahqr works primarily with the Hessenberg submatrix
```

Comment:
Could you add a rough precis of what `ok` actually means. From the list of possible outcomes below, it reads as though some times when `ok` is false, everything is actually OK. Having a conceptual peg to hang things on would help. There are two options, either add here "... of ok[, which indicates whether the factorization converged", or change `ok` to `converged`.